src/foundry/client/pixi/sources/light-source.d.ts

```diff
@@ -213,23 +172,22 @@ declare global {
 
     /**
      * The ratio of dim:bright as part of the source radius
-     * @defaultValue `undefined`
      */
-    ratio: number | undefined;
+    ratio: number;
 
     /**
      * Track which uniforms need to be reset
-     * @internal
      * @defaultValue
-     * ```javascript
+     * ```typescript
      * {
      *   background: true,
      *   illumination: true,
      *   coloration: true
      * }
      * ```
+     * @internal
      */
-    protected _resetUniforms: {
```

Comment:
This and the other underscore prefixed members should stay `protected`.

Reply:
Updated